hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -693,8 +711,22 @@ public String toString() {
     }
 
     final class StsMonitor implements Runnable {
-        private String latestResourceVersion;
-        private RuntimeContext latestRuntimeContext;
+
+        // backoff properties when retrying
+        private static final int MAX_YIELDS = 10;
+        private static final int MAX_PARK_PERIOD_SECONDS = 10;
+
+        // used only for tests
+        volatile boolean running = true;
+        String latestResourceVersion;
+        RuntimeContext latestRuntimeContext;
+        int idleCount;
```

Comment:
```suggestion
        // used only for tests
        volatile boolean running = true;

        String latestResourceVersion;
        RuntimeContext latestRuntimeContext;
        int idleCount;
```

Reply:
done in 77a8077a399212e80268a99ff5d4e9263a4d35b4